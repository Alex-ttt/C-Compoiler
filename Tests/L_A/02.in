4345 5344