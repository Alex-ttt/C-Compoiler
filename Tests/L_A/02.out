1	1	4345    	int number
1	7	5344    	int number
